app/controllers/mixins/ems_common_angular.rb

```diff
@@ -442,9 +443,9 @@ def build_credentials(ems, mode)
         session[:oauth_response] = nil
       end
       if ems.kind_of?(ManageIQ::Providers::ContainerManager) &&
-         ems.supports_authentication?(:bearer) && params[:bearer_password]
-        creds[:hawkular] = {:auth_key => params[:bearer_password], :userid => "_"}
-        creds[:bearer] = {:auth_key => params[:bearer_password]}
+         ems.supports_authentication?(:bearer) && params[:default_password]
+        creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
+        creds[:bearer] = {:auth_key => params[:default_password]}
```

Comment:
> actually this was :bearer_token before... is :default_password better?

in https://github.com/ManageIQ/manageiq/pull/9235 the form fields names (params) changed, from bearer to default. see [issue 9742](https://github.com/ManageIQ/manageiq/issues/9742).

It makes the html/angular form more simple, default is default, and not default is bearer :-)
but it is wrong because in our case default is bearer ...

 > (is it even working?)

It fixes the regression introduced in https://github.com/ManageIQ/manageiq/pull/9235 :-( yes it is warking.